portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Erica Smith's Portfolio</h1>
+      <p>My name is Erica Smith. I'm from Ashland, VA and I'm a rising sophomore at NC A&T SU.
+         This summer, I am a STEP intern with Google.  
+      </p>
+      <img class= "thumbnail align-left"src="/images/aggiesdo.png" alt="AggiesDo">
+      <img class="thumbnail align-right" src="/images/google.png" alt="google">
```

Comment:
Did you mean to add styles for .align-left and .align-right?  If this looks good you can probably remove those classes